x-pack/plugin/ent-search/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/application/FullClusterRestartIT.java

```diff
@@ -24,10 +25,16 @@
 import static org.elasticsearch.Version.V_8_12_0;
 
 public class FullClusterRestartIT extends ParameterizedFullClusterRestartTestCase {
-
+    // DSL was introduced with version 8.1.2.0 of ES.
     private static final Version DSL_DEFAULT_RETENTION_VERSION = V_8_12_0;
 
+    // DSL was introduced with the version 3 of the registry.
+    private static final int DSL_REGISTRY_VERSION = 3;
+
     // Legacy name we used for ILM policy configuration in versions prior to 8.12.0.
+    private static final String EVENT_DATA_STREAM_TEMPLATE_NAME = "behavioral_analytics-events-default";
```

Comment:
Nit: consistency with the naming of `EVENT_DATA_STREAM_LEGACY_ILM_POLICY_NAME`.
```suggestion
    private static final String EVENT_DATA_STREAM_LEGACY_TEMPLATE_NAME = "behavioral_analytics-events-default";
```